src/Stuart/Client.php

```diff
@@ -91,4 +91,15 @@ public function getEta($job)
         $apiResponse = $this->httpClient->performPost($body, '/v2/jobs/eta');
         return json_decode($apiResponse->getBody());
     }
+
+    public function validateAddress($address, $picking)
```

Comment:
I would make this method private, with the following signature: `private function validateAddress($address, $type)`, type being either "picking" or "delivering".

Then I would create two new public methods: `public function validatePickupAddress($address)` and `public function validateDropoffAddress($address)`.

Second option would be to have only one public method `public function validateAddress($address, $type)`, type being either "picking" or "delivering".

WDYT?